.env.sample

```diff
@@ -138,3 +138,6 @@ APP_PORTAL_BASE_URL = ""
 VALIDATE_ACCESS_TOKEN_OFFLINE = "ON/OFF"
 KEYCLOAK_PUBLIC_KEY_PATH = "keycloak-public-keys"
 
+# PUSH SUBMISSION IN IMPROVEMENT SERVICE
+IMPROVEMENT_SERVICE_SUBMISSION_TOPIC = ""
```

Comment:
@aman-tunerlabs - It should be "improvement project"

Reply:
Resolved